src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_open_ephys_recording_interface.py

```diff
@@ -0,0 +1,35 @@
+"""Primary class for converting SpikeGadgets Ephys Recordings."""
+from pynwb.file import NWBFile
+import numpy as np
+from typing import Literal
+
+from neuroconv.datainterfaces import OpenEphysLegacyRecordingInterface
+from spikeinterface.extractors import OpenEphysLegacyRecordingExtractor
+
+
+class Zempolich2024OpenEphysRecordingInterface(OpenEphysLegacyRecordingInterface):
+    """OpenEphys RecordingInterface for zempolich_2024 conversion."""
+
+    Extractor = OpenEphysLegacyRecordingExtractor
+
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+        metadata["Ecephys"]["Device"] = []  # remove default device
+        return metadata
+
+    def add_to_nwbfile(
+        self, nwbfile: NWBFile, metadata: dict, brain_region: Literal["A1", "M2"] = "A1", **conversion_options
+    ):
+        folder_path = self.source_data["folder_path"]
+        channel_positions = np.load(folder_path / "channel_positions.npy")
+        if True:  # TODO: Replace with `if stub_test:` once all channels are present in the data
```

Comment:
How comes you have channels missing?

Reply:
Schneider lab has opted to just include 1 channel/session for quicker/easier data sharing, and then they'll run the conversion themselves with the full data once everything is ready to go.